icons/visualbasic/visualbasic-plain.svg

```diff
@@ -0,0 +1,3 @@
"data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHIAAABRCAYAAAAO5fLlAAAIfUlEQVR4nO2dB7DURByHP88nItiQQceKKNhQUUTs+p4VpdjF3gtWrGMbC2LBMioiImJXHNGxV+yKKCoWxIeCvQ6Wp6CIijxwVn9hjjO5S3K7ubuQb+ZmHlyyu8kvu9n9l72FBrW+gozQrAqsCbQHNgHaAZsDuYICZgJvAN8BE4EfgPeBz1zd6rpMw5KsBfQEdge2CHlOG2APn/9vBF4Cbgbes9nITEh/WgInArtGEC8MnfU5HngHuB0YpR5bFoVDwoJOB2AY8AlwhWURC+kKXAd8DFwCtM2ELB/TA08HPgL6ASsmWPeSwDnAF8AJcQvJhPzv3TcZuBJoUcF2LA4M0SRpo6gnL8hCmqHsUeBBYJUqaI9Hd2A8cGyUkxZUIXcE3gV6V0FbgrgBeCzswQuikOcBo4GVq6AtpegFvBjmQBvLD3ND1gM2BZby+X6iFsZmdvY5MNtCnXG5CziwgvXHoV5iNhQ7N46QphfvDfSRdaN9hHKMoK8DzwJ3A79budTSLAS8poetFqmXIWFbYI5f+6MMra2Aq2RmuhfYH1g94sOwArAncKPKeRzYxfGNNSKOrWERPbYBRgZ9GVbIk7XOOU090AbLyvT1hIaO7rauOA+vJ27moOxKsC9wpF+9pYTsqGHwGhmIXVGv9dOtWk/ZoNaH0yBGAMsXfldMyAZZOrZPsJGHacjtVWY5dSkV0eN/Q2yQkKaHvAAsnEiz5qed1k+jyijj8RSLiDpZz/z/8BOyIezaxTH7aJLSKmI1g4CdqqD9rrklv/xCIc068Jkqaqxx2k4BuoQ83oh4puM2RWWmo3KXA/p6/ygUcnQV+iiNJ2IcsH6J446uMhGNK+xwRRWMdVTHwd4f+UL21QK/GmmpWW3ngLb1AIZXSbvH67XQCbgN+BG4wFFdPfSgzBNyUYUfVDNGzOd92mfeh09VQbt/kud/Y+D+gu+e14Nom5w36fGG0T4W12/GtrqS1nGTgG+BacAs4DcFL5nxvXWEd5+HOe9VLYn+lI33aQc3KCr3yincVOS8wcA9Duo2s/OhdXKm9rdQoHniLgKelNfbCDm9xDldtWY0nw1Dvp+30OhxmvyJleRD4NSQD9NoPXwtLTfYTAiXNOGQPbXuKoeRFrwKxotynIKeWoc4fnqAtyUp7gQOBeZGqG+cg3mIuQ+b5PRiLoezLLmGvgbOBlYDrg1xfKVENCGN2wGHRBTR8ICD9pj70CUXY8HtYcx3GwCXW26YCQ08RZOGVyyXXS5m9rmurF5xeNNRuzbNadyOymR5KyY4vGnj5bo5y2EdYWnSyHVRmeV8APzsoH0d4oR6fK8or98cNMiPyzW7dWUhKYWZyKzts6SIQ1PMjlOKGXGE7J2gZ99jjwqFKh4B7KxFvQ1auHJERDXHmcnIWy4aUoRKxNlMk4DjEq43NlF65JcySifJHRUQ8S29/2tGRNMho/TImxw2xA+zTjso4TqfVk+sNZrD9sjmBIVcRlaQpEU8r0ZFNIypkymtFJNkFHZNGw1tqyV8I/ol5D2Z5Siud7oRcrEQB97moPJC2isyoUMCdXnMkCvIlb+wkHZlGGCCMMuyCTl5K0rxue0rKmAnGaCTFPE9+QyTEhFZwtpYLvNTE7CWU6CTb/RyHktYrjyfkzTJCDMy2MIks3YDpiZYJ0Uc4+VgItD/MkK+rdSyYvzhoAGLyD832EHZxRgoL0tzwvUSJ+8xBP++9rxZaylvg+3g5O3kRdjPcrnFmKik1vMTrDOfFoUhjBaYrvTAeUKOlec9iG6WKm6rqPXn9H5KkvuAhxOuM596B+/Hu7w/8teR/YqcUO4kJKdQiEnKI6kEA5XabXvWGJZi9zcuF+ffYA8z1L0cUOBmCsWIwiJaYA9TGsAQJe5UkhM0yzs84TZ00rBukxHyRP1L4c5Xq5ZYanyiIXiq9okp9JC30BDSTiEN/0s2SYhv5GZbu0h1xsl7roZ51zwE7Gaxjj8V4DYv2KvQ1vqFPByXBRTQUZ9q5lXlXM6QmEHxP92VaWYi4K526NXpaFlEFJg8X8Sen611kC6wFjGhGFtJwLkhZ4n7qneOcrQ8sJ1Hc7efkzvIaN5boRa1ggk52cEnFMO88weEvIZ9dM1G1L0sXfcADYG2aAxyJgQJ+ZcmOLUg5lCZvoLedRdGDOLyIsUblUsSd8Z+jIM16zZBXxRzY82ucjGnyekcZtuvbWM4itfRa+YzhfwfH8Ew0k/7JNikoVgkeyl/5GyFpLvIWyiH4brRgZsjFNCshzKu1988CNcDX2nf1REKTl7D59gBWnLZpEE21UCibLzbP2TgsEtMruSlCgGJg4t9BRoVnPW2clF2tHz9JUUkYvDVYN3I4RXYNWqWBAw7cQlirnIlRlq083oejXpL5XmYGNgDNAKUJGo45FMa0kziyq+WGx7EMN2sckX0mKs9gvrHCPlPipu1FAolIjH3opshw/fq6iWNDi7OmJ7O0BB4nCxKtrlO77hqSrVvUk7JURqFQmNrc/pDFTKxZRmb1k7RpOoRpea58IEG0VdrUL/JSxLM1ix3YNxtsW3/ykCdelE35fwvo1zJdRVl4I0AU5U/8qGSgV7S35XmSI0ESQn6q8I+r1LccGyS+rmIVrJ5ehF7TRXy0Idlb+2Z18NR+t4YRWWY2fcvNgpMagePmRVMwonD/fosrdGlj4zsnWOm6E/QrpiT9dqYYrvB2c9FFGeaTH+e+a+tTHZd9e+tfYLG/paNt1nhJd+7/OEWj0zIaDTp45ktk06jCCT7lYGUkAmZEjIhU0ImZErIhEwJmZApIRMyJWRCpoRMyJSQCZkSMiFTQiZkSsiETAmZkCkhEzIlZEKmhEzIlJAJmRIyIVNCJmTy5EJu5BiJLPgqebzOM0eCzrGRg5IJmTz2E4eAfwAqoo/0w9k79QAAAABJRU5ErkJggg=="/>
```

Comment:
This svg is not a "real" svg... since it contains a raster (png) image. Please provide a real svg without any raster images inside. Also if it can be used as a plain and original icon, call it original :)